cmake/PAL.cmake

```diff
@@ -253,6 +253,11 @@ endforeach()
 
 # set the O3DE_ENGINE_RESTRICTED_PATH
 o3de_restricted_path(${LY_ROOT_FOLDER}/engine.json O3DE_ENGINE_RESTRICTED_PATH)
+if(NOT O3DE_ENGINE_RESTRICTED_PATH)
+    # If the engine.json does not have a 'restricted' field use the engine dir and append /restricted
+    set(O3DE_ENGINE_RESTRICTED_PATH "${LY_ROOT_FOLDER}/restricted")
+    message(STATUS "No restricted path found in engine.json, using default: ${O3DE_ENGINE_RESTRICTED_PATH}")
```

Comment:
Same Q, do we want every user to see this every time?  If its useful for us to see forensically, then its okay.

Reply:
same as above